codecs_derive/src/aper/choice.rs

```diff
@@ -84,7 +84,7 @@ fn generate_choice_variant_decode_tokens_using_attrs(
                             };
                             let variant_encode_token = quote! {
                                 Self::#variant_ident(ref v) => {
-                                    asn1_codecs::aper::encode::encode_choice_idx(data, #lb, #ub, #ext, #key)?;
+                                    asn1_codecs::aper::encode::encode_choice_idx(data, #lb, #ub, #ext, #key, false)?;
```

Comment:
I believe this hard-coded `false` is not a good idea here. We are actually taking the `cp.extended.as_ref` above (line #77), we should simply be using that value?

Reply:
Good point - fixed.